.eslintrc.js

```diff
@@ -135,6 +136,18 @@ module.exports = {
         'jest/valid-expect-in-promise': ERROR,
       },
     },
+    {
+      files: [
+        '**/__tests__/**/*.js',
+        'scripts/**/*.js',
+        'packages/*/npm/**/*.js',
+        'packages/react-devtools*/**/*.js'
```

Comment:
>Maybe we should intentionally exclude those packages, since the component stack logic would never be relevant?

I believe that’s what these lines are doing!

Reply:
Ah, duh. Yeah that would make more sense, now that I look again. okay 👍